webrtc-stats.html

```diff
@@ -216,16 +261,16 @@
     </section><!-- VS: getStats() remains in base spec! -->
     <section>
       <h2>
-        Maintenance procedures for stats types
+        Maintenance procedures for stats object types
```

Comment:
I like it better with "stats object types", because it's sufficiently clear that it's not a stats value type. I guess we differ.